src/GitUtilities.js

```diff
@@ -40,7 +40,7 @@ export default class GitUtilities {
 
   @logger.logifySync
   static pushWithTags(tags) {
-    ChildProcessUtilities.execSync("git push");
+    ChildProcessUtilities.execSync("git push origin " + GitUtilities.getCurrentBranch());
```

Comment:
Yeah, we could attempt to lookup what the tracked remote branch is:

```
$ repo git:(branch): git for-each-ref --format='%(upstream:short)' $(git symbolic-ref -q HEAD)
> origin/master
```

I don't know how reliable that would be though